### PR TITLE
[JENKINS-70052] Delete button in delete build has not enough padding

### PR DESCRIPTION
**Pull request by 1weiho**
See [JENKINS-70052](https://issues.jenkins.io/browse/JENKINS-70052).

Before:
截圖 2022-11-18 上午1 17 53

After:
截圖 2022-11-18 上午1 16 49

### Proposed changelog entries

- Change "Yes" button to "Delete" and add margin between the text and button.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7401"></a>

### CODE REVIEW

**Review by uhafner — APPROVED** [review]

**Review by NotMyFault — COMMENTED** [review]
@uhafner In a recent commit, I reduced the keyboard navigation frame thickness, your screenshot on the issue shows the state before that change.

Do you think the change is still needed with the new design?

**Comment by uhafner** [comment]
> @uhafner In a recent commit, I reduced the keyboard navigation frame thickness, your screenshot on the issue shows the state before that change.
> 

I which Jenkins release is that change? 

> Do you think the change is still needed with the new design?

The dialog is nonsense anyway: it should be a modal dialog rather than a new full layout view.  

I'lll have a look at the latest Jenkins version how it looks like.

**Comment by NotMyFault** [comment]
Starting with 2.378, the frame thickness has been lowered, to no longer overlap components:

Old:
Screenshot 2022-11-19 at 16 49 44
New:
Screenshot 2022-11-19 at 16 50 29

File: core/src/main/resources/hudson/model/Run/confirmDelete.jelly
Comment (janfaracik) [reply core/src/main/resources/hudson/model/Run/confirmDelete.jelly:39]: We could always use one of the spacing classes from https://weekly.ci.jenkins.io/design-library/Spacing/ however it might be worth just having the button on a new line.

**Comment by uhafner** [comment]
It looks better on 2.378, but still not ideal. But I am fine if we want to let this button as it is right now.

I'm still wondering why we did choose two different options for the same thing:

Bildschirm­foto 2022-11-20 um 22 33 26

File: core/src/main/resources/hudson/model/Run/confirmDelete.jelly
Comment (NotMyFault) [reply core/src/main/resources/hudson/model/Run/confirmDelete.jelly:39]: @davidho0403 Could you update your PR please, to use one of the spacings matching, like outlined above, please?

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
